### ISSUE TITLE
Mark nullable fields

### ISSUE THREAD

**Issue by gerbal** [mrin9/RapiDoc#262]
It would be quite helpful if nullable fields could be marked as nullable similar to Redoc 

![Screenshot_2020-07-16 ReDoc](https://user-images.githubusercontent.com/5278092/87706859-305db900-c76e-11ea-92f6-3b23eefcbffd.png)

**Comment by mrin9** [comment]
we are holding onto that change for now, coz nullable is deprecated in OpenAPI v3.1.0
instead the type can be an array `type: [string, "null"]`

We are going to redesign the schema section once v3.1.0 settles down and nulls will be taken care of

**Comment by mrin9** [comment]
Closing this issue for now as it is covered in our schema redesign for 3.1.0 story
Thanks for the suggestion !!!

**Comment by aravindanve** [comment]
@gerbal In case you are still looking for a solution. I have forked this package here https://www.npmjs.com/package/@aravindanve/rapidoc

Which shows nullable primitives like this:
<img width="321" alt="Screenshot 2020-10-07 at 12 59 59 PM" src="https://user-images.githubusercontent.com/5883448/96220728-e8d20d80-0fa6-11eb-842a-c59ae2d57914.png">

And nullable objects like this:
<img width="242" alt="Screenshot 2020-10-07 at 1 00 05 PM" src="https://user-images.githubusercontent.com/5883448/96220726-e7084a00-0fa6-11eb-8af5-41c999aaf348.png">


My docs are consumed by iOS and android clients, and because information is critical in those environments, I had to fork and implement nullable. I'm planning to use this util rapidoc introduces this feature.

@mrin9 let me know if I should send a PR instead

**Comment by jamesvl** [comment]
@aravindanve Do you have a github repo that you keep up to date w/ the main one, but with your changes? I'd love to use it, if so!

Since we don't know when OpenAPI 3.1 will actually be released and face a similar problem where communicating `nullable` is really important.

**Comment by aravindanve** [comment]
@jamesvl I'll update it to the latest version as soon as I get the time. Meanwhile, PRs are welcome https://github.com/aravindanve/rapidoc

**Comment by aravindanve** [comment]
@jamesvl See my comment on your PR https://github.com/aravindanve/RapiDoc/pull/2#issuecomment-775167413

TL;DR v8.4.3 is now available with support for nullable at `@aravindanve/rapidoc@latest` or `@aravindanve/rapidoc@8.4.3-alt.0`
